rust/osp/src/response.rs

```diff
@@ -512,6 +521,47 @@ pub struct Scan {
     pub status: ScanStatus,
     /// Results
     pub results: Results,
+    #[serde(rename = "progress")]
+    /// HostInfo
+    pub host_info: Option<HostInfo>,
+}
+
+/// Information about the scan progress
+#[derive(Clone, Debug, Deserialize, PartialEq)]
+pub struct HostInfo {
+    #[serde(default)]
+    /// Currently scanned hosts
+    pub host: Vec<Host>,
+    /// Overall progress
+    pub overall: ElementU32,
+    /// Number of alive hosts finished
+    /// TODO: Consider divide into alive and finished
```

Comment:
Nit picking: I think the TODO should not be part of the documentation but a normal comment?

Reply:
fixed